### DISCUSSION TITLE
Is it possible to attach a file to the netbox?

### DISCUSSION THREAD

**Discussion by alven-kim** [netbox-community/netbox#17817] [Help Wanted!]
Hello

I want to manage x509 certificate (crt, key) or PDF file in netbox.
accordingly, I would like to ask you the following question.

Q1) Is it possible to attach a file to the netbox?

Q2) When using Type: local, URL: File in the Data Source category, where is the actual PATH of the file on the server?

Thank for your time~!! (__)

**Comment by chris240189** [comment]
for data sources of type local, the URL is just the path on the OS the netbox instance is running on.
for example:
`file:///opt/data-sources/` is the path `/opt/data-sources/` on the server.

**Comment by small75** [comment]
You can use https://github.com/Kani999/netbox-attachments
